plugins/woocommerce-admin/client/core-profiler/actions/tracks.tsx

```diff
@@ -154,11 +154,20 @@ const recordFailedPluginInstallations = (
 	_context: unknown,
 	_event: PluginsInstallationCompletedWithErrorsEvent
 ) => {
+	const failedExtensions = _event.payload.errors.map(
+		( error: PluginInstallError ) => getPluginTrackKey( error.plugin )
+	);
+
 	recordEvent( 'coreprofiler_store_extensions_installed_and_activated', {
 		success: false,
-		failed_extensions: _event.payload.errors.map(
-			( error: PluginInstallError ) => getPluginTrackKey( error.plugin )
-		),
+		failed_extensions: failedExtensions,
+	} );
+
+	failedExtensions.forEach( ( extension ) => {
+		recordEvent( 'coreprofiler_store_extension_installed_and_activated', {
+			success: false,
+			installed_extension: extension,
```

Comment:
Since its not technically always installed, would simply `extension` be more appropriate? The `success` parameter would be responsible for describing how it went.

Reply:
~Do you need to be explicit with the `install_time` time here by setting to `null`?~ Probably not, feel free to ignore

Reply:
> Since its not technically always installed, would simply extension be more appropriate? The success parameter would be responsible for describing how it went.

Sounds good!

Reply:
Changed it to `extension` in a709169abe29dfa0ba0a1ee891f39b55a3357b79